integration-tests/run-books/OCR.md

```diff
@@ -0,0 +1,252 @@
+
+##### Docker
+
+Any test suite/test can be run in Docker using the following `go` command (overrides are automatically injected):
+
+```bash
+go test -v -timeout <max_test_timeout> -p 1 <./path/to/_test.go file>
+```
+
+Example:
+
+```bash
+go test -v -timeout 60m -p 1 ./smoke/ocr2_test.go
+```
+
+##### Kubernetes
+
+Run:
+
+```bash
+BASE64_CONFIG_OVERRIDE=$(cat ./testconfig/overrides.toml | base64) go test -v -timeout <max_test_timeout> -p 1 -run '<TestName>' ./<test_directory>
+```
+
+Example:
+
+```bash
+# Go
+BASE64_CONFIG_OVERRIDE=$(cat ./testconfig/overrides.toml | base64) go test -v -p 1 -run 'TestOCRChaos' ./chaos
+
+- - - - - - - - - - - - - -
+
+# Make
+BASE64_CONFIG_OVERRIDE=$(cat ./testconfig/overrides.toml | base64) make test_soak_ocr
+```
+
+##### CI (with overrides)
+
+For the most tests run [Selected E2E Tests Workflow](https://github.com/smartcontractkit/chainlink/actions/workflows/run-selected-e2e-tests.yml) in GitHub, either manually or using `gh` command unless the otherwise stated:
+
+```bash
+gh workflow run "run-selected-e2e-tests.yml" \
+--ref <branch> \
+-f chainlink_version="v<your.version>" \ # Optional, default is image created from develop branch. Not needed if you run tests against existing environment
+-f workflow_run_name="Any name" \ # Optional
+-f test_ids="<test_ID>" \ # see /chainlink/.github/e2e-tests.yml for IDS
+-f test_secrets_override_key=<your uploaded to GH secrets key> \ # Optional, can be obtained when secrets are uploaded to GitHub
+-f test_config_override_path=<path/to/product/override.toml which should be committed in `integration-tests/testconfig/ocr2/overrides` folder> \ # Optional
+-f with_existing_remote_runner_version=<remote-runner version> \ # Optional
+```
+
+Example:
+
+```bash
+gh workflow run "run-selected-e2e-tests.yml" \
+--ref develop \
+-f chainlink_version="v2.17.0-beta0" \
+-f test_ids="smoke/ocr2_test.go:*" \
+-f workflow_run_name="Smoke:OCR2:2.17.0-beta0" \
+-f test_secrets_override_key=YOUR_TEST_SECRETS_ID \
+-f test_config_override_path=./testconfig/ocr2/overrides/base_sepolia.toml
+```
+
+#### SMOKE
+
+##### Docker
+
+Refer [COMMON COMMANDS#Docker](#docker). Override `<./path/to/_test.go file>` as follows:
+
+With forwarders:
+`./smoke/forwarder_ocr_test.go`
+`./smoke/forwarder_ocr2_test.go`
+
+No forwarders:
+`./smoke/ocr_test.go`
+`./smoke/ocr2_test.go`
+
+##### CI
+
+Refer [COMMON COMMANDS#CI](#ci). Override `test_ids` as follows:
+
+`smoke/ocr_test.go:*` - all OCR1 tests
+`smoke/ocr2_test.go:*` - all OCR2 tests
+
+#### SOAK
+
+> [!IMPORTANT]
+> These tests require logging in to Kubernetes cluster (`aws sso login`).
+> Do not use `-timeout` flag for Soak tests with Go command. It is set in `overrides.toml`
+
+Refer [Test config README](../testconfig/README.md) for more details about Soak tests configuration.
+
+##### Kubernetes
+
+Refer [COMMON COMMANDS#Kubernetes](#kubernetes). Override path as follows:
+
+With forwarders:
```

Comment:
I'd add bullets for each case, it's hard to read this part. Optional though

Reply:
Thanks, updated.